lib/danger/helpers/comments_helper.rb

```diff
@@ -1,10 +1,11 @@
-require "redcarpet"
+# require "redcarpet"
+require "kramdown"
 
 module Danger
   module Helpers
     module CommentsHelper
-      def markdown_parser
-        @markdown_parser ||= Redcarpet::Markdown.new(Redcarpet::Render::HTML.new(hard_wrap: true), no_intra_emphasis: true)
+      def markdown_parser(text)
+        Kramdown::Document.new(text, input: "GFM")
```

Comment:
normally yeap, but Kramdown behaves a bit different to Redcarpet, it needs a string for initialising, so by caching the object it has always the same string to render, think this is not what would be expected